### PR TITLE
Added time elements

### LINKED ISSUE #1499: Add `time` elements on token page
# Description

The token page has multiple places where we display a date.

- Under the title
- When created
- Transaction history

Add a [time elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) for the date.

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- For design guidelines checkout our [Figma design](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap).
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by Ashwani2022**
# Description

[comment]:  Added time HTML tag
Fixes #1499

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/pages/tokens/[tokenid].js [hunk 1]
```diff
@@ -433,7 +443,9 @@ export default function Token(props) {
         <TagList>
           <TreeTag
             key="created-at"
-            TreeTagValue={new Date(token.created_at).toLocaleDateString()}
+            TreeTagValue=<time dateTime={new Date(token.created_at)}>
+              {new Date(token.created_at).toLocaleDateString()}
+            </time>
```
Comment (RubenSmn) [inline src/pages/tokens/[tokenid].js:448]: This prop should be in `{}` also the `dateTime` prop here

```suggestion
            TreeTagValue={
              <time dateTime={token.created_at}>
                {new Date(token.created_at).toLocaleDateString()}
              </time>
            }
```

File: src/pages/tokens/[tokenid].js [hunk 2]
```diff
@@ -315,11 +315,16 @@ export default function Token(props) {
...
                 <Icon icon={CalendarIcon} />
-                {token.created_at !== null
-                  ? `Minted on ${moment(tree.time_created).format(
-                      'MMMM Do, YYYY',
-                    )}`
-                  : 'Unknown Mint Date'}
+                {token.created_at !== null ? (
+                  <>
+                    Minted on
+                    <time dateTime={moment(tree.time_created)}>
```
Comment (RubenSmn) [inline src/pages/tokens/[tokenid].js:321]: No need to pass a moment Date object to the `dateTime` prop the `tree.time_created` is good

```suggestion
                    <time dateTime={tree.time_created}>
```

File: src/pages/tokens/[tokenid].js [hunk 3]
```diff
@@ -369,11 +374,16 @@ export default function Token(props) {
...
```
Comment (RubenSmn) [inline src/pages/tokens/[tokenid].js:380]: Same here with the moment and `dateTime` prop

File: src/pages/tokens/[tokenid].js [hunk 4]
```diff
@@ -433,7 +443,9 @@ export default function Token(props) {
         <TagList>
           <TreeTag
             key="created-at"
-            TreeTagValue={new Date(token.created_at).toLocaleDateString()}
+            TreeTagValue=<time dateTime={new Date(token.created_at)}>
```
Comment (RubenSmn) [inline src/pages/tokens/[tokenid].js:446]: We can just pass `token.created_at` to the `dateTime` prop

```suggestion
            TreeTagValue=<time dateTime={token.created_at}>
```

File: src/pages/tokens/[tokenid].js [hunk 5]
```diff
@@ -504,7 +516,9 @@ export default function Token(props) {
                   flex: '0 0 100px',
                 }}
               >
-                {new Date(token.created_at).toLocaleDateString()}
+                <time dateTime={new Date(token.created_at)}>
```
Comment (RubenSmn) [inline src/pages/tokens/[tokenid].js:519]: `dateTime` prop

File: src/pages/tokens/[tokenid].js [hunk 6]
```diff
@@ -534,7 +548,9 @@ export default function Token(props) {
                   }}
                   color="text.secondary"
                 >
-                  {new Date(transaction.processed_at).toLocaleDateString()}
+                  <time dateTime={new Date(transaction.processed_at)}>
```
Comment (RubenSmn) [inline src/pages/tokens/[tokenid].js:551]: `dateTime` prop

**Review by RubenSmn — CHANGES_REQUESTED** [review]
Some minor issues with the `dateTime` prop, see the comments.

**Comment by Ashwani2022** [comment]
Hi  @RubenSmn 
I have done the changes as suggested.
Can you have a look?
Thanks.

**Review by RubenSmn — APPROVED** [review]
Thanks, awesome job!

**Pull request merged**